### PR TITLE
IP-1880 - Migrate IR RD from 5 to 4

### PR DESCRIPTION
**Pull request by priesgo**
[IP-1880 - Migrate IR RD from 5 to 4](https://jira.extge.co.uk/browse/IP-1880)
===========

Do not merge a test is still failing, please review, I will continue tomorrow.

Additionally:
* Using some factories to create appropriate mocked data
* Fixed an issue on validation of map fields

### CODE REVIEW

File: protocols/migration/participants.py [hunk 1]
```diff
@@ -71,6 +71,73 @@ def _migrate_tumour_sample(self, old_instance):
...
+        return new_instance
+
+    def migrate_ancestries(self, old_instance):
+        new_instance = self.convert_class(self.new_model.Ancestries, old_instance)
+        if old_instance.chiSquare1KGenomesPhase3Pop is not None:
+            new_instance.chiSquare1KGenomesPhase3Pop = [self.migrate_chiblahblah(chi)
+                                                        for chi in old_instance.chiSquare1KGenomesPhase3Pop]
+        return new_instance
+
+    def migrate_chiblahblah(self, old_instance):
```
Comment (greglever) [inline protocols/migration/participants.py]: please don't call it `migrate_chiblahblah`...

File: protocols/tests/test_migration/test_migration_reports_5_0_0_to_reports_4_0_0.py [hunk 2]
```diff
@@ -201,3 +187,34 @@ def test_migrate_rd_clinical_report(self, fill_nullables=True):
     def test_migrate_rd_clinical_report_nullables_false(self):
         self.test_migrate_rd_clinical_report(fill_nullables=False)
 
+    def test_migrate_rd_interpretation_request(self, fill_nullables=True):
+        # creates a random clinical report RD for testing filling null values
```
Comment (greglever) [inline protocols/tests/test_migration/test_migration_reports_5_0_0_to_reports_4_0_0.py]: this comment is only True if fill_nullables is True

**Comment by greglever** [comment]
otherwise looks good

**Comment by priesgo** [comment]
@greglever re-review

**Pull request merged**